assets/src/components/editing/toolbar/editorToolbar/EditorToolbar.tsx

```diff
@@ -27,6 +29,9 @@ export const EditorToolbar = (props: Props) => {
       <BlockToggle blockInsertOptions={props.insertOptions} />
       <BlockSettings />
       <BlockInsertMenu blockInsertOptions={props.insertOptions} />
+      {props.onSwitchToMarkdown && (
```

Comment:
Note: Availability of the new menu is dependent on an event handler for it being set. No need for two params.